util/data_stream.go

```diff
@@ -33,8 +33,8 @@ var validTypes = map[string]string{
 	"metrics": "Metrics",
 }
 
-type Dataset struct {
-	// Name and type of the dataset. This is linked to dataset.name and dataset.type fields.
+type DataStream struct {
+	// Name and type of the dataStream. This is linked to dataStream.name and dataStream.type fields.
 	Type string `config:"type" json:"type" validate:"required"`
 	Name string `config:"name" json:"name,omitempty" yaml:"name,omitempty"`
```

Comment:
`name` should be `dataset` following https://github.com/elastic/package-spec/pull/24/files#diff-13d2900dcce8e312a06c834b665816a1R71

Reply:
Thanks for finding it! I'm looking into this.

Reply:
Fixed